storage/client/queries/queries.go

```diff
@@ -278,11 +278,31 @@ const (
 		LIMIT $2::bigint
 		OFFSET $3::bigint`
 
-	Validators = `
+	LatestEpochStart = `
 		SELECT id, start_height
 			FROM chain.epochs
 			ORDER BY id DESC`
 
+	ValidatorsAggStats = `
+		WITH delegators AS (
+			SELECT delegator FROM chain.delegations
+			GROUP BY delegator
+		)
+		SELECT
+			COALESCE (
+				(SELECT SUM(voting_power)
+				FROM chain.nodes)
+			, 0) AS total_voting_power,
+			COALESCE (
+				(SELECT COUNT(*)
```

Comment:
Could use `SELECT COUNT(distinct delegator) from chain.delegations`?

It might be a bit clearer than using a CTE?

Reply:
good idea; updated!